trademarks.md

```diff
@@ -0,0 +1,144 @@
+ When the trademark no longer represents a certain level of quality to the community,
+ or no longer indicates that we are the source of the products that bear the trademark,  
+ the trademark loses its value.
+
+Underlying Project Jupyter's trademark policy is a set of guidelines for what is -- and is not -- acceptable use of our trademarks, specifically the word mark "Jupyter", the Jupyter logos, and variations of those marks.
+This policy describes the uses generally approved by Jupyter and NumFOCUS for its trademarks.
+However, if you violate this policy, or otherwise take actions that may compromise the reputation or trademarks of Jupyter, or expose NumFOCUS to liability, NumFOCUS may require you to cease all use of any Jupyter trademark, regardless of the uses allowed in this policy.
+
+
+## General Goals
+
+In general, we want the word mark "Jupyter" and the Jupyter logos to be used with minimal restriction to refer to Jupyter events, software, or services.
+
+We do not want these trademarks to be used:
+
+- to refer to any other events, software, or services
+- in a way that is misleading or may imply association of third-party events, software, or services with Project Jupyter
+- in ways that confuse the community as to whether the Project Jupyter is open source and free to use
+
+
+## Uses that Never Require Approval
+
+All trademarks are subject to "nominative use rules" that allow use of the trademark to name the trademarked entity in a way that is minimal and does not imply a sponsorship relationship with the trademark holder.
+
+As such, stating accurately that software or a service integrates with Jupyter software, that it uses Jupyter software, that it is compatible with Jupyter software, or that it contains Jupyter software, is always allowed.
+In those cases, you may use the word "Jupyter" or the unaltered logos to indicate this, without our prior approval.
+This is true both for non-commercial and commercial uses.
+
+This clause overrides other clauses of this policy.
+However, if you have any doubts about your intended use of the trademarks, please contact the [Jupyter Trademarks Committee][].
+
+
+### Use of the Jupyter trademark on merchandise
+
+We encourage use of the Jupyter trademarks (logo and word mark) on stickers, hats, mugs, t-shirts and other physical merchandise to promote the project and the broader open-source ecosystem.
+No explicit approval is needed for these uses as long as
+1) the use follows the official Jupyter Brand Guidelines and
+2) the merchandise is provided free of charge.
+We understand that not everyone (including ourselves) can afford to give away merchandise in sufficient quantities to meet demand.
+Because of this we would love help from the community in creating an official place to purchase Jupyter branded
+merchandise with the profits going back to the project through NumFOCUS.
+If you are interested in helping out with this please get in touch with us on the Jupyter Google Group.
```

Comment:
Make it a [link to the google group](https://groups.google.com/forum/#!forum/jupyter)